common/middleware/instrument.go

```diff
@@ -39,14 +40,32 @@ func (i Instrument) Wrap(next http.Handler) http.Handler {
 	})
 }
 
+var invalidChars = regexp.MustCompile(`[^a-zA-Z0-9]+`)
```

Comment:
Have tried to reverse engineer reasoning here; put a comment in https://github.com/weaveworks/scope/pull/1570/commits/ae39936a571783ba066042a7357495d6ad7f0a2e.  Does this sound about right?